app/Http/Controllers/v1/ProviderPlatformController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers\v1;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\StoreProviderPlatformRequest;
+use App\Http\Requests\UpdateProviderPlatformRequest;
+use App\Http\Resources\PaginateResource;
+use App\Http\Resources\ProviderPlatformResource;
+use App\Models\ProviderPlatform;
+
+class ProviderPlatformController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        $providerPlatforms = ProviderPlatform::paginate(10);
+
+        return PaginateResource::make($providerPlatforms, ProviderPlatformResource::class);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
```

Comment:
I know these are generated but I believe I removed them from the User one after our meeting as they aren't following the separation of concerns we outlined. not a big deal but they can be removed